README.md

```diff
@@ -122,19 +121,19 @@ Questions on how to add your app to Groot or use the Groot API:
     source ~/.profile
     nvm install 7.2.0
     nvm use 7.2.0
-    sudo npm install -g forever
```

Comment:
Do you need this for just working on the project (btw will probably rename this repo to groot after the arbor changes are done)